### PR TITLE
Added wrapper to allow by CJS loading

### PR DESCRIPTION
**Pull request by euangoddard**

### CODE REVIEW

File: socket.js [hunk 1]
```diff
@@ -4,100 +4,99 @@
...
-        var prefix = options.prefix === undefined ? defaultPrefix : options.prefix ;
-        var defaultScope = options.scope || $rootScope;
-
-        var addListener = function (eventName, callback) {
-          socket.on(eventName, callback.__ng = asyncAngularify(socket, callback));
-        };
-
-        var addOnceListener = function (eventName, callback) {
-          socket.once(eventName, callback.__ng = asyncAngularify(socket, callback));
+(function(window, angular, undefined) {'use strict';
```
Comment (btford) [inline socket.js]: Wrapping this is an unrelated change– if you're using CJS, this is especially redundant because your build tool already does it. Can you revert this change?

**Comment by euangoddard** [comment]
Removed the wrapper. Thanks for the feedback
